### PR TITLE
Add 2006-rebotted

### PR DESCRIPTION
**Pull request by dginovker**
Open source 2006 Runescape Emulation with a focus on the reality of how the game was played back then, i.e. there were tons of bots.

### CODE REVIEW

**Comment by dginovker** [comment]
Possibly failed because I created a new file? (`2.yaml`)

**Comment by cxong** [comment]
please move the changes into `0.yaml`

File: games/0.yaml [hunk 1]
```diff
@@ -44,6 +44,29 @@
   updated: 2019-04-15
   url: https://kilgoretroutmaskreplicant.gitlab.io/plain-html/
 
+- name: 2006-rebotted
+  originals:
+    - RuneScape (2006 era)
```
Comment (cxong) [inline games/0.yaml]:
```suggestion
    - Runescape Classic
```

**Comment by dginovker** [comment]
Does it not like the numbers at all?

Runescape Classic is 2001 era, this is a 2006 era recreation :P

**Comment by cxong** [comment]
build errors were in master, should be fixed now

**Pull request merged**